scania-digital-design-system/sdds
# Enabling tooltip to wrap HTML code

**Describe pull-request**  
Adding slot to Tooltip component so it can wrap other HTML content

**Solving issue**  
Fixes: #[AB1022](https://dev.azure.com/scaniadigitaldesignsystem/SDDS/_workitems/edit/1022)

**How to test**  
1. Checkout the branch/PR
2. Go to tooltip component
3. In story files, try to add additional HTML in between <sdds-tooltip... ></sdds-tooltip> and see if it renders out
